src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,80 @@
+<!--Please create here a HTML example by using just default HTML tags-->
+
+<button onclick="openModal()">Open modal</button>
+
+<axa-modal class="axa-modal-preview">
+  <h2>Prämie berechnen</h2>
+  <p><strong>Mehr als 80%</strong> unserer Kunden kombinieren die Hausratversicherung mit einer Privathaftpflichtversicherung.</p>
+  <p>Was möchten Sie?</p>
+
+  <section class="grid">
+    <div>
+      <img class="huge" src="/assets/icons/rocking_chair_blue.svg" />
```

Comment:
please use `axa-icon` instead

Reply:
I cannot "blackbox-style" it, at least I did not find a way. I don't want to go and change the styling by accessing the internal classes of that axa-icon, because as soon as those internal css classes change, my design would break with it. Since this is only a preview, I think it's better to not have an additional dependency there from here with this situation. But this is probably worth a discussion.